app/util/workflows.ts

```diff
@@ -121,3 +123,30 @@ export async function terminateWorkflows(job: Job, logger: Logger): Promise<void
     throw e;
   }
 }
+
+/**
+ *
+ * Check if it's a turbo workflow for a given Job
+ *
+ * @param jobID - The job id associated with the workflow
+ * @param transaction - the transaction to use for querying
+ * @param logger - The Logger to use for logging errors/info
+ */
+export async function checkIfTurboWorkflow(
+  transaction: Transaction,
+  jobID: string,
+  logger: Logger,
+): Promise<boolean> {
+  let isTurboWorkflow = false;
+  try {
+    const hasWorkItemsTable = await transaction.schema.hasTable('work_items');
+    if (hasWorkItemsTable) {
+      const workItemCount = await workItemCountForJobID(transaction, jobID);
+      if (workItemCount) isTurboWorkflow = true;
```

Comment:
while this works because 0 is falsey, it probably is easier to follow if we treat workItemCount as a number:
```
if (workItemCount > 0) isTurboWorkflow = true;
```